crates/trie/sparse/src/trie.rs

```diff
@@ -759,7 +982,16 @@ impl<P> RevealedSparseTrie<P> {
         self.rlp_node(prefix_set, &mut buffers)
     }
 
-    /// Look up or calculate the RLP of the node at the given path specified in [`RlpNodeBuffers`].
+    /// Recursively looks up or computes the RLP encoding of the node specified by the current
```

Comment:
This method doesn't have recursion in its traditional sense, let's remove it as it can be confusing